core/templates/pages/blog-dashboard-page/blog-dashboard-page.mainpage.html

```diff
@@ -11,6 +11,16 @@
     }
     </script>
     <style>
+      mat-tooltip-component .mat-tooltip.oppia-mat-tooltip-list {
+        font-size: 12px;
+        white-space: pre-wrap;
+      }
+
+      button:focus {
+        border: unset;
+        outline: unset;
+      }
+
```

Comment:
Please make this rule more targeted.

Reply:
Since we need to remove the border effects of angular mat components, adding any other class forces to use `!important`